api/tsconfig.json

```
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "module": "node16",
+    "target": "ES2022",
+    "declaration": true,
+    "sourceMap": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "moduleResolution": "nodenext",
+    "skipLibCheck": true,
+    "allowJs": false,
+    "noImplicitAny": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "templates"]
```

I cannot find the directory 'templates' in api repository
```suggestion
  "exclude": ["node_modules"]
```